exir/tensor.py

```diff
@@ -66,8 +68,17 @@ def dim_order_from_stride(stride: Tuple[int]) -> Tuple[bytes]:
     for _, s in enumerate(stride):
         if s == 0:
             raise ValueError("0 in strides is not supported for ExecuTorch.")
+
+    def cmp(a, b):
+        if guard_size_oblivious(a[1] < b[1]):
+            return -1
+        elif guard_size_oblivious(a[1] > b[1]):
```

Comment:
Hmm, I am suspicious. Can you show me that this equivalent to this preexisting code?

```
    # Checks that there exists a permutation of the strides s.t. the tensor would be contiguous
    # Sorts (length, stride) pairs by stride
    #
    # This sort is done in a size-oblivious way, which helps if we do a
    # comparison like 2048*u0 > u0; we just want this to return True
    # (and not worry about what if u0 is zero).
    class K(NamedTuple):
        size: int
        stride: int

        def __lt__(self, other):
            return guard_size_oblivious(self.stride < other.stride)
        
        def __gt__(self, other):
            return guard_size_oblivious(self.stride > other.stride)
    
        def __le__(self, other):
            return guard_size_oblivious(self.stride <= other.stride)
    
        def __ge__(self, other):
            return guard_size_oblivious(self.stride >= other.stride)

        def __eq__(self, other):
            return guard_size_oblivious(self.stride == other.stride)

```

Otherwise, use something like this.

Reply:
Thanks for the pointer! Didn't realize that already existed. 

Will close this PR however, as we will take a different approach as discussed internally.